sample_apps/simple_app_under_gramine/gramine_example_app.mak

```diff
@@ -38,6 +38,7 @@ SGX_LDFLAGS = -Wl,--enable-new-dtags
 
 ######################### CERTIFIER ###########################
 CERTIFIER_SRC_PATH = ../../src
+COMMON_SRC = ../common
```

Comment:
Same comment as previously stated.

Could you move to specifying dirs off of CERTIFIER_ROOT; i.e. this becomes

COMMON_SRC = $(CERTIFIER_ROOT)/sample_apps/common